### DISCUSSION TITLE
Hard crash on song startup

### DISCUSSION THREAD

**Discussion by Mof10** [ppy/osu#27044] [Q&A]
### Type

Crash to desktop

### Bug description

I logged in and tried to play a song, after which osu crashed. Not entirely sure where else to report this.

### Version

2024.131.0-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14159164/compressed-logs.zip)

1707093105.runtime.log seems to have the most relevant output.

**Comment by bdach** [comment]
Logs are cut off abruptly which likely indicates a higher level crash.

Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.

**Comment by Mof10** [comment]
Attaching the event log (in a .zip archive) as requested. it should be noted that the crash occurred at approx. 4:35PM local time on my machine.
[event_log.zip](https://github.com/ppy/osu/files/14162520/event_log.zip)

**Reply by Mof10** [reply]
Also, after looking through the event logs, I located the relevant .wer file relating to the crash.
[AppCrash_osu!.exe_e36c4c9aebb5b6c8f51d9f22c61eb1e115bd16_f96185e8_c7ffc45d-514c-4a33-b05e-f20b7fa81917.zip](https://github.com/ppy/osu/files/14162575/AppCrash_osu.exe_e36c4c9aebb5b6c8f51d9f22c61eb1e115bd16_f96185e8_c7ffc45d-514c-4a33-b05e-f20b7fa81917.zip)

**Reply by bdach** [reply]
```
Application: osu!.exe
CoreCLR Version: 6.0.2623.60508
.NET Version: 6.0.26
Description: The process was terminated due to an internal error in the .NET Runtime at IP 00007FFB41993C89 (00007FFB417C0000) with exit code c0000005.
```

Please begin by checking for graphics card driver updates and installing them if they're available.

**Reply by Mof10** [reply]
Yep, there's an updated driver from 1/24 that I didn't install, I presume that's probably the culprit?

**Reply by bdach** [reply]
Well it's one thing to try. If it doesn't help there are others we can try, but this is the usual first step that helps the most people.

**Reply by Mof10** [reply]
Will do! I'll update if the crash happens again, but it doesn't seem to have occurred since then.
